wrappers/python-scrapy/{projectFolder}/__main__.template.py

```diff
@@ -13,11 +13,15 @@
 # `scrapy.utils.log` before we patch it.
 import logging
 from typing import Any
-import scrapy.utils.log
+from scrapy.utils import log as scrapy_logging
+from scrapy.utils.project import get_project_settings
 from apify.log import ActorLogFormatter
 
-# If you want to change the logging level, change it here
-LOGGING_LEVEL = logging.INFO
+# To change the logging level, modify the `LOG_LEVEL` field in `settings.py`. If the field is not present in the file,
+# Scrapy will default to `DEBUG`. This setting applies to all loggers. If you wish to change the logging level for
+# a specific logger, do it in this file.
```

Comment:
Does this mean that for freshly wrapped projects, the log level will always be `DEBUG`?  I assume that a regular `settings.py` in a random Scrapy project doesn't include this field, right?

Reply:
> Does this mean that for freshly wrapped projects, the log level will always be DEBUG?

Not really, the value from the user's `settings.py` will be taken. 

> I assume that a regular `settings.py` in a random Scrapy project doesn't include this field, right?

It depends. `LOG_LEVEL` is a standard settings field - https://docs.scrapy.org/en/latest/topics/settings.html#log-level. So users can set it up. We follow the Scrapy defaults here.

Reply:
Ah okay then, no further objections :)